shopper-app/src/views/Account.tsx

```diff
@@ -0,0 +1,326 @@
+            description="View all transactions, manage payments methods and settings"
+            alt="Payments"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Your Amazon Family"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/account._CB660668669_.png"}
+            description="Manage profiles, sharing, and permissions in one place"
+            alt="Family"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Digital Services and Device Support"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/digital_devices._CB660668735_.png"}
+            description="Troubleshoot device issues, manage or cancel digital subscriptions"
+            alt="Digital-services"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Archived orders"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/fshub/10_archived_orders._CB654640573_.png"}
+            description="View and manage your archived orders"
+            alt="Archived-orders"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Your Lists"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/fshub/11_lists._CB654640573_.png"}
+            description="View, modify, and share your list, or create new ones"
+            alt="Lists"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Customer Service"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/contact_us._CB659962323_.png"}
+            description="Browse self service options, help articles or contact us"
+            alt="Customer"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4}>
+          <AccountCards
+            title="Your Messages"
+            image={"https://m.media-amazon.com/images/G/01/x-locale/cs/help/images/gateway/self-service/fshub/9_messages._CB654640573_.jpg"}
+            description="Manage your account security settings"
+            alt="Messages"
+            onClick={() => handleNavigation('comingSoon')}
+          />
+        </Grid>
+      </Grid>
```

Comment:
150+ lines of repeated code. Try putting this stuff into a list like your other code